_test/tests/inc/mailer.test.php

```diff
@@ -88,27 +88,49 @@ function test_addresses(){
         $mail->to('Andreas Gohr <andi@splitbrain.org>');
         $mail->cleanHeaders();
         $headers = $mail->prop('headers');
-        $this->assertEquals('Andreas Gohr <andi@splitbrain.org>', $headers['To']);
+
+        if (isWindows()) { // see FS#652
+            $this->assertEquals('andi@splitbrain.org', $headers['To']);
+        } else {
+            $this->assertEquals('Andreas Gohr <andi@splitbrain.org>', $headers['To']);
+        }
 
         $mail->to('Andreas Gohr <andi@splitbrain.org> , foo <foo@example.com>');
         $mail->cleanHeaders();
         $headers = $mail->prop('headers');
-        $this->assertEquals('Andreas Gohr <andi@splitbrain.org>, foo <foo@example.com>', $headers['To']);
+        if (isWindows()) { // see FS#652
+            $this->assertEquals('andi@splitbrain.org,  foo@example.com', $headers['To']);
+        } else {
+            $this->assertEquals('Andreas Gohr <andi@splitbrain.org>, foo <foo@example.com>', $headers['To']);
+        }
 
         $mail->to('Möp <moep@example.com> , foo <foo@example.com>');
         $mail->cleanHeaders();
         $headers = $mail->prop('headers');
-        $this->assertEquals('=?UTF-8?B?TcO2cA==?= <moep@example.com>, foo <foo@example.com>', $headers['To']);
+
+        if (isWindows()) { // see FS#652
+            $this->assertEquals('moep@example.com,  foo@example.com', $headers['To']);
+        } else {
+            $this->assertEquals('=?UTF-8?B?TcO2cA==?= <moep@example.com>, foo <foo@example.com>', $headers['To']);
+        }
```

Comment:
I think for tests which's task is testing something that does not work/exist in Windows, it would make more sense to just mark the test as skipped. Eg. here the idea is to test the encoding in the name, but since the name is omitted on windows, there is nothing to test that hasn't been tested above already.